google-cloud-firestore/src/main/java/com/google/cloud/firestore/FirestoreImpl.java

```diff
@@ -213,11 +247,26 @@ public void onCompleted() {
       final Transaction.Function<T> transactionCallback,
       final SettableApiFuture<T> resultFuture,
       final TransactionOptions options) {
+    Span span = tracer.spanBuilder("CloudFirestore.Transaction").startSpan();
```

Comment:
Please comment that the "span" is intentionally not ended here and that it will be ended if an error occurs or success.

Reply:
Done.